bin/pycbc_multi_inspiral

```diff
@@ -68,25 +66,29 @@ parser.add_argument(
 )
 parser.add_argument("--bank-file", type=str)
 parser.add_argument(
-    "--snr-threshold", type=float, help="SNR threshold for trigger generation"
-)
-parser.add_argument(
-    "--newsnr-threshold",
+    "--snr-threshold",
     type=float,
     metavar='THRESHOLD',
-    help="Cut triggers with NewSNR less than THRESHOLD",
+    help="Single IFO SNR threshold for trigger generation."
 )
+# Commenting out unused options: remove if they remain unused
+# parser.add_argument(
+#     "--newsnr-threshold",
+#     type=float,
+#     metavar='THRESHOLD',
+#     help="Cut triggers with NewSNR less than THRESHOLD",
+# )
```

Comment:
I will remove this in 2 PRs from now.  I have one coming up for further cleanup to this executable and then one on arguments that will affect these lines.

Reply:
Fair enough.